code/modules/clothing/suits/f13factionarmor.dm

```diff
@@ -189,14 +189,12 @@
 	icon_state = "legion-opifex"
 	item_state = "legion-opifex"
 	blood_overlay_type = "armor"
-	armor = list("melee" = 5,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 10, "acid" = 5)
-	allowed = list(/obj/item/crowbar,
-		/obj/item/screwdriver,
-		/obj/item/weldingtool,
-		/obj/item/wirecutters,
-		/obj/item/wrench,
-		/obj/item/stack/cable_coil,
-		/obj/item/twohanded/sledgehammer)
+	allowed = list(	/obj/item/crowbar,
+					/obj/item/screwdriver,
+					/obj/item/weldingtool,
+					/obj/item/wirecutters,
+					/obj/item/wrench,
+					/obj/item/twohanded)
```

Comment:
```suggestion
	allowed = list(
		/obj/item/crowbar,
		/obj/item/screwdriver,
		/obj/item/weldingtool,
		/obj/item/wirecutters,
		/obj/item/wrench,
		/obj/item/twohanded,
		)
```
* Use trailing commas and put the closing parentheses on a different line, to prevent errors and diminish diffs.
* Avoid unnecessary indentation / horizontal drift. Remember that different editors might not show the same spacing.